mne/source_estimate.py

```diff
@@ -1366,6 +1367,41 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         return brain
 
     @verbose
+    def return_to_original_src(self, src_orig, subject_orig=None,
```

Comment:
I don't like this name. Maybe

remap_to_src ?
map_to_src ?
map_to ?

cc @dengemann @teonlamont anyone ?